experimental/PropertyDDS/packages/property-dds/src/propertyTree.ts

```diff
@@ -157,25 +164,24 @@ export class SharedPropertyTree extends SharedObject {
 		this._root.cleanDirty(BaseProperty.MODIFIED_STATE_FLAGS.DIRTY);
 	}
 
-	public get changeSet() {
-		// eslint-disable-next-line @typescript-eslint/no-unsafe-return
+	public get activeCommit(): CommitNode {
 		return this.tipView;
```

Comment:
This is misleading. The active commit is not the tipView. It would be this.localChanges[this.localChanges.length -1] (or the last entry in remoteChanges if there are no local changes).

Reply:
Yeah that is confusing I will revert this a little more and make a separate activeCommit.